### PR TITLE
Removes the deltastation prison automapper template

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request
As Jolly reminded me today, this template is fucking huge and a maintainer headache (I would know I have to maintain it) for how little to the round it actually provides, and how little importance it actually has. As such, no more!

## How This Contributes To The Skyrat Roleplay Experience
I (The person who makes your maps work) will no longer have to spend several hours with every big code change fixing this stupid goddamn prison.

## Changelog

:cl:
del: The automapper delta prison has been removed, we will instead just use the default tg prison
/:cl:

### CODE REVIEW

**Review by Jolly-66 — APPROVED** [review]
YES
YES
YES
YES

File: _maps/skyrat/automapper/automapper_config.toml [hunk 1]
```diff
@@ -100,14 +100,6 @@ required_map = "DeltaStation2.dmm"
 coordinates = [121, 124, 1]
 trait_name = "Station"
```
Comment (Zonespace27) [inline _maps/skyrat/automapper/automapper_config.toml:101]:
```suggestion
trait_name = "foobar"
```
checks are fucked and I want it to rerun

**Comment by Paxilmaniac** [comment]
ok

**Comment by press-mark** [comment]
ripbozo

**Comment by Zonespace27** [comment]
100% maptainer approval (prisoner mains malding)

**Pull request merged**